tests/integration/test_split.py

```diff
@@ -0,0 +1,343 @@
+# Copyright 2021-2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import math
+
+import numpy as np
+import pytest
+
+import cunumeric as num
+
+# cunumeric.split(a: ndarray, indices: Union[int, ndarray], axis: int = 0)
+# → list[cunumeric.array.ndarray]
+# cunumeric.vsplit(a: ndarray, indices: Union[int, ndarray])
+# → list[cunumeric.array.ndarray]    (axis=0)
+# cunumeric.hsplit(a: ndarray, indices: Union[int, ndarray])
+# → list[cunumeric.array.ndarray]    (axis=1)
+# cunumeric.dsplit(a: ndarray, indices: Union[int, ndarray])
+# → list[cunumeric.array.ndarray]    (axis=2)
+
+# Seed the random generator with a random number
+np.random.seed(416)
```

Comment:
I think @manopapad wants to handle this differently, see 

* https://github.com/nv-legate/cunumeric/pull/792

Reply:
Got it.